### PR TITLE
added descrption of context variables

### PR DESCRIPTION
**Pull request by puja108**
added the 4 context variables and changed some wording

### CODE REVIEW

**Comment by puja108** [comment]
RFR @marians @ewoutp

**Comment by puja108** [comment]
regarding https://github.com/giantswarm/docs-content/issues/89

**Comment by marians** [comment]
I guess I would turn the use the other way around: `myexample-$GIANTSWARM_LOGIN_NAME`.

Then, in the explanation of the context variables, we could mention the "complete example" to point users to a way how to use these variables, like this: "In our 'Complete example' above, we use the $GIANTSWARM_LOGIN_NAME to add the current user's username into the domain entry."

Besides that, LGTM.

**Comment by puja108** [comment]
In the firstapp guides we always use yourusername-currentweather so I thought of using the same order here, too.

Btw, we should maybe adapt the firstapp guides to use these variables instead of the make ones. (will create Issue/PR)

Agrre with the addition of an explanation. Will do

**Comment by puja108** [comment]
added example eplanation

**Comment by puja108** [comment]
@marians do you want to have another look?

File: content/reference/swarm-json.md [hunk 1]
```diff
@@ -327,13 +327,24 @@ or
...
+## Giant Swarm context variables
+
+There are four Giant Swarm context variables available for use in your `swarm.json`. The respective values of these varialbes are set based on your login and current selected environment. Note that you cannot change the values of these variables unless you change your actual login or environment through CLI or API.
+
+* GIANTSWARM_ORGANIZATION: e.g. "giantswarm"
+* GIANTSWARM_ENVIRONMENT: e.g. "dev"
+* GIANTSWARM_ENV: e.g. "giantswarm/dev"
+* GIANTSWARM_LOGIN_NAME: e.g. "yourusername"
+
+You can use these anywhere in your `swarm.json`. E.g. in our ['Complete example'](#complete-example) above, we use the $GIANTSWARM_LOGIN_NAME variable to customize the domain entry of the application based on the current user's username.
```
Comment (marians) [inline content/reference/swarm-json.md]: I would set $GIANTSWARM_LOGIN_NAME in backticks here (code/verbatim string)

File: content/reference/swarm-json.md [hunk 2]
```diff
@@ -327,13 +327,24 @@ or
...
 * `path` (optional): If specified, this will be used as mounting point of the volume from the referenced component. If not specified, the mounting point will be equal to the mounting point of the referenced component.
 
 <i class="fa fa-exclamation-triangle"></i> Please note that we currently do not provide a backup mechanism. If you need to preserve the data on your volumes, please think about a solution using for example an FTP server or cloud storage like Amazon S3 from within your component.
 
-## Making use of configuration variables
+## Giant Swarm context variables
+
+There are four Giant Swarm context variables available for use in your `swarm.json`. The respective values of these varialbes are set based on your login and current selected environment. Note that you cannot change the values of these variables unless you change your actual login or environment through CLI or API.
+
+* GIANTSWARM_ORGANIZATION: e.g. "giantswarm"
```
Comment (marians) [inline content/reference/swarm-json.md]: Here backticks, too

**Comment by marians** [comment]
Little formatting thing... then LGTM

File: content/reference/swarm-json.md [hunk 3]
```diff
@@ -327,13 +327,24 @@ or
...
 
-## Making use of configuration variables
+## Giant Swarm context variables
+
+There are four Giant Swarm context variables available for use in your `swarm.json`. The respective values of these varialbes are set based on your login and current selected environment. Note that you cannot change the values of these variables unless you change your actual login or environment through CLI or API.
+
+* `GIANTSWARM_ORGANIZATION`: Your current selected organization, e.g. "giantswarm"
+* `GIANTSWARM_ENVIRONMENT`: Your current selected environment, e.g. "dev"
+* `GIANTSWARM_ENV`: Your current selected organization and environment, e.g. "giantswarm/dev"
+* `GIANTSWARM_LOGIN_NAME`: Your username that you logged in with, e.g. "yourusername"
```
Comment (ewoutp) [inline content/reference/swarm-json.md]: Logged in where? This is not clear.

It should be "Your username that you logged in with into Giant Swarm ..."

File: content/reference/swarm-json.md [hunk 4]
```diff
@@ -327,13 +327,24 @@ or
...
+
+You can use these anywhere in your `swarm.json`. E.g. in our ['Complete example'](#complete-example) above, we use the `$GIANTSWARM_LOGIN_NAME` variable to customize the domain entry of the application based on the current user's username.
```
Comment (ewoutp) [inline content/reference/swarm-json.md]: You may consider adding a note here that it is not the local machine username.

File: content/reference/swarm-json.md (hunk 3 above)
Comment (puja108) [inline content/reference/swarm-json.md]: ok, will clarify this

**Pull request merged**